### DISCUSSION TITLE
Custom DB

### DISCUSSION THREAD

**Discussion by matze1610** [prisma/prisma#6317] [Q&A] [answered]
Hello community,
is there a way to add additional databases to Prisma 2? Right now I only found a "documentation will follow" remark, but no place where I could look further. 

Thanks
  Mathias

**Comment by ryands17** [comment]
@matze1610 👋 
By additional databases if you mean adding another database to the schema, then this is not possible in a single schema, but you can use multiple schemas as described in a comment here: https://github.com/prisma/prisma/issues/2443#issuecomment-630679118

**Reply by matze1610** [reply]
I wasn't aware that this is possible, very helpful - thanks a lot! Next thing to read on ;) But indeed I meant to use a DB not supported yet, sorry for not being clear!
Thanks for your help
  Mathias

**Comment by janpio** [comment]
What exactly are you trying to do?
Do you want to support for other databases, that are not supported yet?
Where did you find this remark, can you provide a link?

**Reply by matze1610** [reply, marked as answer]
Yes, I just wanted to check what to do if a DB is not yet supported. I'm sorry, I've already read the sign "you've reached the end of the internet" ;) I do not remember where I read this. 
Use case: we are going to use PostGres for the time being, but we expect that we have to support other DBs as well - Sybase Anywhere and Hana are candidates, but even Object Store is in the talks. 

Thanks for your time
  Mathias

**Reply by janpio** [reply]
Then currently the correct answer is: Create an issue for us to take it into consideration. The codebase is not _yet_ set up for contributions of new connectors for additional databases. If you can refind that line you mentioned, I would update it to be more in this direction.

**Reply by matze1610** [reply]
Thanks for the answer, Jan, I will follow that advice. 
And yes, I'll keep looking for the site!
Happy easter
  Mathias

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
